lib/AST/AvailabilityContext.cpp

```diff
@@ -170,6 +174,10 @@ bool AvailabilityContext::isUnavailableInEmbedded() const {
   return Info->Platform.IsUnavailableInEmbedded;
 }
 
+bool AvailabilityContext::allowsUnsafe() const {
+  return Info->Platform.AllowsUnsafe;
```

Comment:
It could be good to add this bit to the AvailabilityScope dump output for debugging.

Reply:
Great idea, done in the last commit. Thanks!